lambdas/services/dynamo_service.py

```diff
@@ -82,6 +96,18 @@ def update_item_service(self, table_name, key, update_expression, expression_att
             logger.error(e)
             raise e
 
+    def delete_item_service(self, table_name: str, key: dict):

```

Comment:
Same here, the function belongs to a service and so is implemented `DynamoDBService.delete_item_service` doesn't really make sense to have `service` mentioned twice? Could we maybe rename these `service` functions to just be what the function is doing?

Reply:
Done